test/services/FinancialInstitutionsServiceSpec.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import base.SpecBase
+import connectors.FinancialInstitutionsConnector
+import models.FinancialInstitutions.TINType.GIIN
+import models.FinancialInstitutions.{AddressDetails, ContactDetails, FIDetail, TINDetails}
+import org.mockito.MockitoSugar.when
+import org.scalatestplus.mockito.MockitoSugar._
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class FinancialInstitutionsServiceSpec extends SpecBase {
+
+  val mockConnector: FinancialInstitutionsConnector = mock[FinancialInstitutionsConnector]
+  val sut                                           = new FinancialInstitutionsService(mockConnector)
+
+  implicit override val hc: HeaderCarrier = HeaderCarrier()
+
+  "FinancialInstitutionsService" - {
+
+    "getListOfFinancialInstitutions extracts list of FI details" in {
+      val subscriptionId = "XE5123456789"
+      val mockResponse   = Future.successful(HttpResponse(200, viewFIDetailsBody))
+
+      when(mockConnector.viewFis(subscriptionId)).thenReturn(mockResponse)
+      val result: Future[Seq[FIDetail]] = sut.getListOfFinancialInstitutions(subscriptionId)
+      result.map {
+        fiDetails =>
+          fiDetails mustBe Seq(
```

Comment:
can this simply be `result.futureValue mustBe ...` ?

Reply:
it can indeed! 👍